packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -219,243 +227,251 @@ class AccountSwitcher extends React.Component {
 
         const total_assets_message = this.is_real_account_tab ? total_assets_message_real : total_assets_message_demo;
 
-        return (
-            <div className='acc-switcher__list' ref={this.setWrapperRef}>
-                <Tabs
-                    active_index={this.state.active_tab_index}
-                    className='acc-switcher__list-tabs'
-                    onTabItemClick={this.updateAccountTabIndex}
-                    top
-                >
-                    {/* TODO: De-couple and refactor demo and real accounts groups
-                        into a single reusable AccountListItem component */}
-                    <div label={localize('Real')}>
-                        <ThemedScrollbars
-                            autoHeight
-                            autoHide
-                            autoHeightMax={this.props.is_mobile ? 400 : 354}
-                            renderTrackHorizontal={props => (
-                                <div {...props} className='track-horizontal' style={{ display: 'none' }} />
-                            )}
-                            renderThumbHorizontal={props => (
-                                <div {...props} className='thumb-horizontal' style={{ display: 'none' }} />
-                            )}
+        const demo_accounts = (
+            <div className='acc-switcher__list-wrapper'>
+                <div>
```

Comment:
is this extra `<div>` necessary?

Reply:
@kelly-binary , seems like it was there before, I merely copied it when refactoring, I'll try removing it to see if breaks anything.

Reply:
@kelly-binary , replaced `<div>..</div>` with `<>..</>` and it seems like it's all good.